pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+build-backend = "flit_core.buildapi"
+requires = ["flit_core >=3.8.0,<4"]
+
+[project]
+name = "ydata-sdk"
+authors = [
+    {name = "YData", email = "opensource@ydata.ai"},
+]
+description = "Python package developer's cheat sheet (using pyproject.toml)."
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+]
+dynamic = [
+    "version",
+]
+dependencies = [
+    "numpy==1.21.*",
```

Comment:
hopefully we will not need this 😂  well at least not for now, so leave it empty please